.github/workflows/test.yml

```
@@ -68,7 +67,8 @@ jobs:
         run: etc/bin/shellchecker
 
       - name: ESLint
-        run: yarn run lint
+        run: |
+          npx --package=eslint -- eslint -c .eslintrc.js sphinx/
```

Would have to be v8 max
```suggestion
          npx --package=eslint@8.x -- eslint -c .eslintrc.js sphinx/
```